master/buildbot/steps/vstudio.py

```diff
@@ -198,27 +198,27 @@ def finished(self, result):
 
 class VC6(VisualStudio):
 
-    default_installdir = 'C:\\Program Files\\Microsoft Visual Studio'
+    default_installdir = r'C:\Program Files\Microsoft Visual Studio'
```

Comment:
I used r"" as @sa2ajj requested so that all the slashes in path strings can be single instead of double.